tests/Api/ServiceTest.php

```diff
@@ -538,6 +538,34 @@ public function transactionService()
         $this->assertInstanceOf(Entities\Transaction::class, $result);
     }
 
+    /**
+     * @test
+     */
+    public function payoutService()
+    {
+        $client = new Client(['apiKey' => 'QWERTY']);
+
+        /** @var CurlHandler|MockObject $handler */
+        $handler = $this->createMock(CurlHandler::class);
+
+        $handler
+            ->expects($this->any())
+            ->method('__invoke')
+            ->will($this->returnValue(
+                $client->createResponse()->withHeader('Location', 'payouts')
+            ));
```

Comment:
This is incorrect code. The POST request returns resources with Location to `/transactions/{id}`